src/components/Password/ToggleButton.ts

```diff
@@ -0,0 +1,47 @@
+import { variant } from 'styled-system';
+import styled from 'styled-components';
+
+import { get } from '../../utils/themeGet';
+import { InputProps } from '../Input/InputProps';
+
+const inputType = variant({
+    prop: 'variant',
+    variants: {
+        'bottom-lined': {
+            justifyContent: 'right'
+        }
+    }
+});
+
+const ToggleButton = styled.button<Pick<InputProps, 'size' | 'variant'>>`
+    box-sizing: border-box;
+    position: absolute;
+    background: transparent;
+    border-radius: ${get('radii.2')};
+    border: none;
+    cursor: pointer;
+    display: inline-flex;
+    justify-content: center;
+    text-align: center;
+    text-decoration: none;
+    height: 100%;
+    width: 2.5rem;
```

Comment:
Shouldn't we try to relate this to the padding 2.5rem we have within `PasswordWrapper`, perhaps a constant file?

Reply:
Done